public/apps/configuration/constants.tsx

```diff
@@ -19,6 +19,7 @@ export const API_ENDPOINT = API_PREFIX + '/configuration';
 export const API_ENDPOINT_ROLES = API_PREFIX + '/configuration/roles';
 export const API_ENDPOINT_ROLESMAPPING = API_PREFIX + '/configuration/rolesmapping';
 export const API_ENDPOINT_ACTIONGROUPS = API_PREFIX + '/configuration/actiongroups';
+export const API_ENDPOINT_SECURITYCONFIG = API_PREFIX + '/configuration/securityconfig';
```

Comment:
I think we can derive from API_ENDPOINT rather than from API_PREFIX. Will send a PR to update Line 19~21 also.

Reply:
Let me know how you plan to proceed it. Either I can wait for your PR to merged or you can update this API along with your PR.

Reply:
Nvm I'll update them together later.